docs/monitoring.md

```diff
@@ -312,15 +312,39 @@ can be identified by their `[attempt-id]`. In the API listed below, when running
     <td>Details for the storage status of a given RDD.</td>
   </tr>
   <tr>
-    <td><code>/applications/[base-app-id]/logs</code></td>
-    <td>Download the event logs for all attempts of the given application as files within
+    <td><code>/applications/[app-id]/logs</code></td>
+    <td>Download the event logs of the given application as files within
```

Comment:
The previous version is more correct. It's also unrelated to streaming, so you probably shouldn't change this.